crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -171,14 +185,8 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
-    /// Creates a new entity with the components contained in `bundle`.
-    ///
-    /// This returns an [`EntityCommands`] builder, which enables inserting more components and
-    /// bundles using a "builder pattern".
-    ///
-    /// Note that `bundle` is a [`Bundle`], which is a collection of components. [`Bundle`] is
-    /// automatically implemented for tuples of components. You can also create your own bundle
-    /// types by deriving [`derive@Bundle`].
```

Comment:
Why was this removed? Just asking for the rationale. In my opinion it removes additional valuable info, but I might miss context etc.

Reply:
I removed them because they are out of context descriptions that belong to `EntityCommands` and `Bundle` respectively. If the reader doesn't know their definition, they can just follow the link.

Reply:
Ok makes sense!